pandas/tests/window/test_rolling.py

```diff
@@ -394,6 +417,7 @@ def test_rolling_datetime(axis_frame, tz_naive_fixture):
     tm.assert_frame_equal(result, expected)
 
 
+# maybe this? center true/false
```

Comment:
Yes if you could test center here too that'd be great.

Reply:
added the `center=True` path in the same way now, and used the center fixture for it :)